solr/core/src/test/org/apache/solr/handler/admin/ZookeeperStatusHandlerTest.java

```diff
@@ -74,6 +78,39 @@ public void tearDown() throws Exception {
     super.tearDown();
   }
 
+  @Test
+  public void testZkread() throws Exception {
+    URL baseUrl = cluster.getJettySolrRunner(0).getBaseUrl();
+    String basezk = baseUrl.toString().replace("/solr", "/api") + "/cluster/zk";
+
+    try(  HttpSolrClient client = new HttpSolrClient.Builder(baseUrl.toString()).build()) {
+      Object o = Utils.executeGET(client.getHttpClient(),
+          basezk + "/security.json",
+          Utils.JSONCONSUMER );
+      assertNotNull(o);
+      o = Utils.executeGET(client.getHttpClient(),
+          basezk + "/configs",
+          Utils.JSONCONSUMER );
+      assertEquals("0", String.valueOf(getObjectByPath(o,true, split(":/configs:_default:dataLength",':'))));
+      assertEquals("0", String.valueOf(getObjectByPath(o,true, split(":/configs:conf:dataLength",':'))));
+      byte[] bytes = new byte[1024*5];
+      for (int i = 0; i < bytes.length; i++) {
+        bytes[i] = (byte) random().nextInt(128);
```

Comment:
why make things so complicated? cant you just put some static text? what are you trying to achieve with this randomization?

Reply:
this was the easiest way to create a `byte[]` .  could use anything else. But this code was simpler

Reply:
Wait, can't you do `"this is some test content".getBytes(StandardCharsets.UTF_8);`?

Reply:
This is still unresolved. Can you clarify why we need this random byte array for the test?

Reply:
I wanted a big enough `byte[]` not a small one.